android/src/main/java/com/thebluealliance/androidclient/models/Match.java

```diff
@@ -356,7 +357,6 @@ public MatchListElement render(boolean showVideo, boolean showHeaders, boolean s
         try {
             videos = getVideos();
         } catch (FieldNotDefinedException e) {
-            Log.w(Constants.LOG_TAG, "Required field for match render: Database.Matches.VIDEOS. Defaulting to none.");
```

Comment:
Weird request... can we leave this in for now? Eventually I'll see this span from the ticker fragment and have the time to get to the bottom of the performance issue with the ticker rendering (even if this isn't the root of the performance problem) :stuck_out_tongue: